lib/Dialect/MSFT/ExportQuartusTcl.cpp

```diff
@@ -199,6 +205,19 @@ LogicalResult TclOutputState::emit(PDRegPhysLocationOp locs) {
   return success();
 }
 
+LogicalResult TclOutputState::emit(PDMulticycleOp op) {
+  indent() << "set_multicycle_path ";
+  os << "-hold 1 ";
```

Comment:
Previous we emitted the hold and setup on different lines of SDC.   Have you checked that this works with quartus?

Reply:
I have not, but based on [the documentation](https://www.intel.com/content/www/us/en/docs/programmable/683432/21-4/tcl_pkg_sdc_ver_1.5_cmd_set_multicycle_path.html) i see no reason how this should not work.